src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -112,7 +109,7 @@ def from_dict(cls, config_dict) -> "AnalysisConfig":
             std_cutoff=config_dict.get(ConfigKeys.STD_CUTOFF_KEY, 1e-6),
             stop_long_running=config_dict.get(ConfigKeys.STOP_LONG_RUNNING, False),
             global_std_scaling=config_dict.get(ConfigKeys.GLOBAL_STD_SCALING, 1.0),
-            max_runtime=max_runtime,
+            max_runtime=config_dict.get(ConfigKeys.MAX_RUNTIME, 0),
```

Comment:
this change (together with the deleted lines 104-106 above) to how `MAX_RUNTIME` is picked from a config dict seems unrelated to my eyes  - would you mind shortly explaining this change? is it just tidying up, because we do not expect `MAX_RUNTIME` to be a list in the config dict?

Reply:
> is it just tidying up, because we do not expect MAX_RUNTIME to be a list in the config dict?

Yes. 

Before the change, I was forced to add ln 104-106 and to guard against the possibility of users adding multiple instances of `MAX_RUNTIME` in the same config file. And "there can be only one" (movie quote) `MAX_RUNTIME` and now `config_content.as_dict()` makes sure this is the case.

Reply:
ok cool, now i understand, thanks!